notebook/static/terminal/js/main.js

```diff
@@ -29,13 +29,6 @@ requirejs([
     var common_config = new configmod.ConfigSection('common', common_options);
     common_config.load();
 
-    var login_widget = new loginwidget.LoginWidget('span#login_widget', common_options);
```

Comment:
Please leave this in place - it makes the 'logout' button in the top right work.